src/main/java/cloud/fogbow/ras/core/plugins/authorization/DistributedAuthorizationPlugin.java

```diff
@@ -0,0 +1,147 @@
+package cloud.fogbow.ras.core.plugins.authorization;
```

Comment:
We should talk about this design next meeting. I am not following, so it is hard to comment.

Reply:
Updated the design, as discussed.